src/components/app/details/cdDetails/DeploymentDetailSteps.tsx

```diff
@@ -54,19 +58,38 @@ export default function DeploymentDetailSteps({ deploymentStatus, deploymentAppT
         }
     }, [])
 
-    return deploymentStatus === 'Aborted' ? (
+    const redirectToDeploymentStatus = () => {
+        const newUrl = `${URLS.APP}/${appId}/${URLS.APP_DETAILS}/${envId}${URLS.DEPLOYMENT_STATUS}`
+        history.push(newUrl)
+    }
+
+    return deploymentStatus === 'Aborted' || deploymentStatusDetailsBreakdownData.deploymentStatus === 'superseded' ? (
         <div className="flexbox deployment-aborted">
             <CDEmptyState
-                title="This deployment was aborted"
-                subtitle="This deployment was aborted as a successive deployment was triggered before this deployment could complete."
+                title="Deployment failed"
+                subtitle="A new deployment was initiated before this deployment completed."
             />
         </div>
     ) : deploymentListLoader ? (
         <Progressing pageLoader />
+    ) : deploymentStatusDetailsBreakdownData.deploymentStatusBreakdown.APP_HEALTH.isCollapsed === false ? (
```

Comment:
Can't we use  `!deploymentStatusDetailsBreakdownData.deploymentStatusBreakdown.APP_HEALTH.isCollapsed`instead of deploymentStatusDetailsBreakdownData.deploymentStatusBreakdown.APP_HEALTH.isCollapsed === false